org.eclipse.xtext/src/org/eclipse/xtext/formatting2/debug/TextRegionsToString.java

```diff
@@ -88,6 +88,8 @@ protected String quote(String string, int maxLength) {
 			case '\r':
 				if (!ignoreCarriageReturnInQuotes) {
```

Comment:
maybe invert the conditions for better readability?

Reply:
Conditions are now swapped.